packages/kbn-handlebars/index.ts

```diff
@@ -747,6 +875,9 @@ class ElasticHandlebarsVisitor extends Handlebars.Visitor {
         );
       }
 
+      // inherit partials from parent program
+      runtimeOptions.partials = runtimeOptions.partials || this.runtimeOptions!.partials;
```

Comment:
Yeah, it's because `runtimeOptions` isn't set in the `constructor`, but in `render`, which technically means it's possible to get to this point in the code before it's set. But only theoretically as `render` is ALWAYS called before anything else, so we can just treat it as if it is always set.

We use `this.runtimeOptions!` all over the code, not only in this function, so I would prefer an approach that satisfied the entire class than to simply add a local variable in each function. Otherwise I prefer the `!` approach.